### PR TITLE
Add hardware tutorials from wpilib.org

### PR DESCRIPTION
**Pull request by ItayZiv**
Add's the hardware tutorials from https://wpilib.org/videos to the docs

### CODE REVIEW

File: source/docs/hardware/hardware-tutorials/motors.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Motors for robotics applications
```
Comment (Daltz333) [inline source/docs/hardware/hardware-tutorials/motors.rst]: ```Motors for Robotics Applications```

File: source/docs/hardware/hardware-tutorials/sensors.rst [hunk 2]
```diff
@@ -0,0 +1,27 @@
+Sensing and sensors
```
Comment (Daltz333) [inline source/docs/hardware/hardware-tutorials/sensors.rst]: Sensing and Sensors

**Comment by ItayZiv** [comment]
I added a disclaimer, should the direct links of the playlists also be added for completeness? 
Also not super sure if the admonition is note or if it should be something else.

File: source/docs/hardware/hardware-tutorials/index.rst [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Hardware Tutorials
+==================
+
+.. note:: The pages in this sections of the documentation contain media which is only viewable from the web version of the documentation
```
Comment (Daltz333) [inline source/docs/hardware/hardware-tutorials/index.rst:4]:
```suggestion
.. note:: The pages in this section of the documentation contain media which is only viewable from the web version of the documentation
```

**Pull request closed without merging**